structured_data/2021_07_02_TFDF_serving/TF-DF-serving.md

```diff
@@ -0,0 +1,50 @@
+# Serving TF-DF models using TF Serving
+
+*This is a small excerpt from the  https://blog.ml6.eu/serving-decision-forests-with-tensorflow-b447ea4fc81c blogpost*
+
+TensorFlow and TensorFlow Extended are a broad ecosystem of software packages, allowing to train, serve,... ML models.
+
+The addition of decision forests to TensorFlow (blogpost)[https://www.tensorflow.org/decision_forests], increases the applicability of TensorFlow on many more use cases.
+
+Training is only one part of ML models: robust inference calculations are as important in a data pipeline.
+This is where TensorFlow Serving comes in. TensorFlow Serving basically creates an API around a model. On top of that it allows to split code and model, enabling separate versioning.
+
+Unfortunately, when TensorFlow Decision Forests (TF-DF) was released, TensorFlow Serving was not ready to except these new models.
+
+As we found TF-DF models to be very interesting, we, at ML6, decided to build a custom docker image to serve these models.
+
+The docker image can be found at (https://hub.docker.com/r/ml6team/tf-serving-tfdf)[https://hub.docker.com/r/ml6team/tf-serving-tfdf].
+
+To use the docker image for a penguin model, the following code can be used:
```

Comment:
Maybe refer to the colab that was used to train this penguin model?

Reply:
has been added